config/docker-monitor-filelog.json

```diff
@@ -9,6 +9,7 @@
 	"lookback": "5m",
 	"bufferSize": 10,
 	"source": "docker-monitor",
+	"skipList": [],
```

Comment:
No need to add empty list in existing files. We need to make sure when it is missing, those files still work for backward compatibility.

Reply:
done